Fix #19267 - Fix SVG dimensions reset to 0

liviuconcioiu opened the pull request:
### Description

`width` and `height` in svg were reset to 0.

Fixes #19267 

Before:

https://github.com/user-attachments/assets/f60034bd-b8ee-4e16-b46d-7eca78e404ef

After:

https://github.com/user-attachments/assets/81aa7137-9ba7-4d4c-9bbf-5738f360b669

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

liviuconcioiu committed:
Fix SVG dimensions reset to 0

Signed-off-by: Liviu-Mihail Concioiu <liviu.concioiu@gmail.com>

liviuconcioiu commented:
@MoonE would you like to look at this and test please? Thank you!

MoonE commented:
The demo server is kind of out of date, last version is from january. I imported the table locally, but it is working fine for me in both Firefox and Chrome.

liviuconcioiu committed:
Fallback to original svg dimensions

Signed-off-by: Liviu-Mihail Concioiu <liviu.concioiu@gmail.com>

liviuconcioiu commented:
On localhost I can reproduce in Chrome too. So definitely not a browser bug. I think it depends how fast the modal is open, it can't get the svg size.

I reverted the initial change and added a fall back to the original svg dimensions.
